### ISSUE TITLE
Agenda : Dans l'espace privé, l'agenda devient uniquement attribuable à telle rubrique

### ISSUE THREAD

**Issue by touti** [QuartiersXXI/quartiersxxi#9]
Pour permettre un code générique, éviter ainsi une erreur d'attribution d'agenda à une rubrique qui génère un mauvais affichage public.

**Comment by rastapopougros** [comment]
Sur le même principe que la demande pour GIS, si je comprends bien ? C'est-à-dire configurer dans quelle(s) branche(s) on verrait les boutons ?

**Comment by touti** [comment]
Oui, c'est cela. D'autant que le plugin Agenda sur ce projet est réservé à un seul article qui regroupe les évènements.
